File: workspace/showHideForObjC/showHideForObjC/ViewController.m
//
//  ViewController.m
//  showHideForObjC
//
//  Created by snowman on 2017/02/10.
//  Copyright © 2017年 snowman. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIImageView *flowerImgV;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (IBAction)buttonAction:(UIButton *)sender {
    
    _flowerImgV.hidden = !_flowerImgV.hidden;
}


@end

File: workspace/showhide/showhide/ViewController.swift
//
//  ViewController.swift
//  showhide
//
//  Created by yoshiyuki oshige on 2016/07/22.
//  Copyright © 2016年 yoshiyuki oshige. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var flower: UIImageView!
    
    @IBAction func showhide(_ sender: AnyObject) {
        flower.isHidden = !flower.isHidden
        print(flower.isHidden)
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

